lib/cocoapods-core/yaml_helper.rb

```diff
@@ -143,6 +145,8 @@ def process_array(array)
       # @return [String] the YAML representation of the given object.
       #
       def process_hash(hash, hash_keys_hint = nil, line_separator = "\n")
+        return '{}' if hash.empty?
```

Comment:
this breaks "empty" lockfiles, so we might need to add a test for empty lockfiles explicitly?

Reply:
no it doesn't, since we always add the CocoaPods version. The change in the integration spec is expected (and correct), since we're manually removing the cocoapods version (to avoid noisy diffs)